src/styles/calendar.scss

```
@@ -0,0 +1,34 @@
+.calendar {
+  margin: 100px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $calendarGap;
+  padding: $calendarPadding;
+  width: $calendarMaxWidth;
+
+  &__day {
+    height: $cellHeight;
+    box-sizing: border-box;
+    width: $cellHeight;
+    position: relative;
+    border: 1px solid black;

```

don't forget to create variables for all colors
```suggestion
    border: 1px solid $c-black;

```